e2e/test/scenarios/dashboard/reproductions/29304-scalar-first-render-size-change.cy.spec.js

```diff
@@ -0,0 +1,120 @@
+import { restore } from "e2e/support/helpers";
+
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+
+// Couldn't import from `metabase/components/ExplicitSize` because dependency issue.
+// It will fail Cypress tests.
+const WAIT_TIME = 300;
```

Comment:
Can't import from `metabase/components/ExplicitSize` because it will also import other dependencies in that file and will cause [Cypress test to crash](https://github.com/metabase/metabase/actions/runs/5656362887/job/15323756506?pr=32534).